.chloggen/gen-marshal-json-stream.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: pdata
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use pdatagen to generate marshalJSON without using gogo proto jsonpb.
```

Comment:
We should probably mention that fields with zero values aren't marshalled anymore